test-apps/display-performance-test-app/src/backend/backend.ts

```diff
@@ -36,17 +37,19 @@ export async function initializeBackend() {
 
   if (ProcessDetector.isElectronAppBackend) {
     const rpcInterfaces = [DisplayPerfRpcInterface, IModelTileRpcInterface, SnapshotIModelRpcInterface, IModelReadRpcInterface];
+    const authClient = await ElectronAuthorizationBackend.create({
+      clientId: "imodeljs-electron-test",
```

Comment:
Can you switch this back to the environment variables?

Reply:
Should be fixed.